### ISSUE TITLE
[bug]Regularly experiencing OOM running node even with 24GB RAM

### ISSUE THREAD

**Issue by xinzhongyoumeng** [NibiruChain/nibiru#1934]
# Basic information
* Version: v1.4.0
*  Deployment mode: docker
* OS: Ubuntu 22.04.3 LTS
# Description
I'm regularly experiencing OOM running node even with 24GB RAM，found there is a bug in cosmos wasmvm:https://github.com/CosmWasm/cosmwasm/issues/1978
Is there any chance of getting nibiru update with fixed version of CosmWasm?

**Comment by Unique-Divine** [comment]
Yes, we have that coming up in one of the next sprints for #1868. I'll communicate to the other devs that we should prioritize this.

**Comment by xinzhongyoumeng** [comment]
Is there any progress？
